en/deploy-on-alibaba-cloud.md

```diff
@@ -197,6 +197,10 @@ All the instances except ACK mandatory workers are deployed across availability
 
         Modify `replicas`, `storageClaims[].resources.requests.storage`, and `storageClassName` according to your needs.
 
+        > **Warning:**
+        >
+        > Since TiDB Operator will mount PVs automatically in the **order** of the items in the `storageClaims` list, if you need to add more disks to TiFlash, please make sure to only append the new item to the **end** of the original items, and **DO NOT** modify the order of the original items.
```

Comment:
```suggestion
        > Since TiDB Operator will mount PVs automatically in the **order** of the items in the `storageClaims` list, if you need to add more disks to TiFlash, make sure to append the new item only to the **end** of the original items, and **DO NOT** modify the order of the original items.
```